src/main/java/io/crate/jmx/recorder/NodeStatus.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.jmx.recorder;
+
+import io.prometheus.client.Collector;
+
+import java.util.Collections;
+
+public class NodeStatus implements Recorder {
+
+    static final String MBEAN_NAME = "NodeStatus";
+
+    @Override
+    public boolean recordBean(String domain,
+                              String attrName,
+                              Number beanValue,
+                              MetricSampleConsumer metricSampleConsumer) {
+        if (attrName.equalsIgnoreCase("Ready")) {
+            String fullname = domain + "_ready";
```

Comment:
should we extract the hardcoded strings into constants so they don't get out of hand? eg "READY_ATTRIBUTE_NAME" and "READY_DOMAIN_SUFFIX"